apps/site-landing/src/containers/HelpDocs/index.tsx

```diff
@@ -26,48 +25,24 @@ const HelpDocsInternal = (props: RouteComponentProps) => {
   }
 
   useEffect(() => {
-    actions.fetchFaqs();
+    actions.fetchAllDocs();
   }, []);
 
   const categories = buildCategories(docs);
-  useEffect(() => {
-    sidebarActions.addGenerator(
-      FAQS_KEY,
-      () => {
-
-        const items: SidebarMenuItem[] = [
-
-          ...Object.keys(categories)
-            .sort()
-            .map(category => (
-              {
-                link: {
-                  name: category,
-                  to: buildUrl(category)
-                }
-              }
-            ))
-        ]
-        return items;
-      });
-    return () => sidebarActions.removeGenerator(FAQS_KEY);
-  }, [docs]);
-
+  console.log("categories",categories)
   return (
-    <Switch>
-      {
-        Object.entries(categories)
-          .map((entry, index) => {
-            const url = buildUrl(entry[0]).toString()
-            return <Route key={index} path={url} render={() => <FAQs faqs={entry[1]} />} />
-          })
-        // docs.faqs.map(a =>
-        //   <Route key={a.id} path={buildUrl(a.id).toString()} render={()=> <Article {...a} />} />
-        // )
-        //<Route path={buildUrl("").toString()} exact={true} render={()=> <FAQs faqs={docs.faqs} />} />
-      }
-      <Route path={buildUrl("").toString()} exact={true} render={()=> <Welcome faqs={docs.faqs} />} />
-    </Switch>
+    <>
+      <Switch>
+        {
+          Object.entries(categories)
+            .map((entry, index) => {
+              const url = buildUrl("theme-"+((entry[0])?.split("-"))[0].replace(/ /g,'')).toString()
```

Comment:
similar comment to above.  I -think- this is the same select, but it would be nice for it to be de-duplicated